code/game/objects/items/weapons/alien_specific.dm

```diff
@@ -19,14 +19,14 @@
 
 //Bottles for borg liquid squirters. PSSH PSSH
 /obj/item/reagent_containers/spray/alien
-	name = "liquid synthesizer"
-	desc = "squirts alien liquids."
```

Comment:
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/41c002b0-045e-4043-8b22-5073f3a13205)

Reply:
And good riddance!